includes/net-standard-table.md

```diff
@@ -1,4 +1,4 @@
-| .NET Standard              | [1.0]  | [1.1]  | [1.2] | [1.3] | [1.4] | [1.5]              | [1.6]              | [2.0]               | [2.1] <br /> Preview |
+| .NET Standard              | [1.0]  | [1.1]  | [1.2] | [1.3] | [1.4] | [1.5]              | [1.6]              | [2.0]               | [2.1] <br /> |
 |----------------------------|--------|--------|-------|-------|-------|--------------------|--------------------|---------------------|---------------------
 | .NET Core                  | 1.0    | 1.0    | 1.0   | 1.0   | 1.0   | 1.0                | 1.0                | 2.0                 | 3.0 |
 | .NET Framework <sup>1</sup>| 4.5    | 4.5    | 4.5.1 | 4.6   | 4.6.1 | 4.6.1 <sup>2</sup> | 4.6.1 <sup>2</sup> | 4.6.1 <sup>2</sup>  | N/A<sup>3</sup> |
```

Comment:
@terrajobst once you know the TBD values for the .NET Standard 2.1 table, please let me know and I'll update this PR

Reply:
Unity told us they are about 2 years out before they update the Mono dependencies, so that won't be any time soon. With respect to UWP, that depends on Windows and their annuncements, for which I don't know the timelines for. So we're likely shipping .NET Core 3.0/.NET Standard 2.1 with TBD entries.

Reply:
OK, good to know! Thanks.